mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -2117,6 +2128,7 @@ impl From<TakerOrder> for MakerOrder {
                 base_orderbook_ticker: taker_order.base_orderbook_ticker,
                 rel_orderbook_ticker: taker_order.rel_orderbook_ticker,
                 p2p_privkey: taker_order.p2p_privkey,
+                db_id: None, // TODO,
```

Comment:
Should be the same as the taker order `db_id`. This is used to convert timed out taker orders to maker orders.